src/main/scala/uk/gov/hmrc/ct/ct600/v2/RSQ1.scala

```diff
@@ -22,4 +22,3 @@ import uk.gov.hmrc.ct.ct600.v2.retriever.ReturnStatementsBoxRetriever
 case class RSQ1(value: Option[Boolean]) extends CtBoxIdentifier with CtOptionalBoolean with Input with ValidatableBox[ReturnStatementsBoxRetriever] {
   override def validate(boxRetriever: ReturnStatementsBoxRetriever): Set[CtValidation] = validateAsMandatory(this)
```

Comment:
we should use the new SelfValidatableBox instead of ValidatableBox in this case, so we can avoid passing a reference to the box, i.e.

validateAsMandatory()